tests/cql/CqlListOperatorsTest.xml

```diff
@@ -852,49 +872,27 @@
 		</test>
 		<test name="UnionListNullAndListNull">
 			<expression>{ null } union { null }</expression>
-			<output>{null, null}</output>
+			<output>{null}</output>
```

Comment:
According to the spec:
> The union operator is defined as a set operation, meaning that duplicates are eliminated; if an element appears in both sources, that element will only appear once in the resulting list.

and:
> This operator uses equality semantics with the exception that null elements are considered equal.

So nulls should be collapsed to a single null.